core/config/locales/en.yml

```diff
@@ -1115,6 +1115,8 @@ en:
     hints:
       spree/price:
         country: "This determines in what country the price is valid.<br/>Default: Any Country"
+        master_variant: "Changing these prices will not change variant prices below, but will be used to populate all new variants"
+        options: "These are the options used in the table below. They can be changed in the variants tab"
```

Comment:
The new bootstrap layouts have a place for this kind of information where we could also put links. Let's leave it in the tooltip for now, and when we convert over to the new layouts we can augment the text with a link in the sidebar.